src/WorkflowStep.spec.ts

```diff
@@ -291,12 +291,12 @@ describe('WorkflowStep', () => {
 
   describe('processStepMiddleware', () => {
     it('should call each callback in user-provided middleware', async () => {
-      const { next: _next, ...fakeArgs } = createFakeStepEditAction() as unknown as AllWorkflowStepMiddlewareArgs;
+      const { ...fakeArgs } = createFakeStepEditAction() as unknown as AllWorkflowStepMiddlewareArgs;
       const { processStepMiddleware } = await importWorkflowStep();
 
       const fn1 = sinon.spy((async ({ next: continuation }) => {
         // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
I think we can now safely remove this line as well.

Reply:
Thanks! Fixed